src/components/BondingCurveChart.tsx

```diff
@@ -0,0 +1,426 @@
+            hasActiveInput && futureSupply.gt(initGoal);
+
+        // If future supply is within the kickstarter phase, show it there
+
+        console.log('chartParams', {
+            datParams: datStore.datParams[configStore.activeDatAddress],
+            initReserve: initReserve.toString(),
+            initGoal: initGoal.toString(),
+            buySlopeNum: buySlopeNum.toString(),
+            buySlopeDen: buySlopeDen.toString(),
+            currentSupply: totalSupplyWithoutPremint.toString(),
+            hasInitGoal: hasInitGoal,
+            hasExceededInitGoal: hasExceededInitGoal,
+            points: points,
+        });
+
+        /*
+            Generate Kickstarter section content
+         */
+        if (hasInitGoal && !hasExceededInitGoal) {
+            datasets.push(
+                generateLine(
+                    [points.currentSupply, points.kickstarterEnd],
+                    chartGreen,
+                    'Kickstarter Funded'
+                )
+            );
+
+            datasets.push(
+                generateLine(
+                    [points.kickStarterStart, points.currentSupply],
+                    chartBlue,
+                    'Kickstarter Unfunded'
+                )
+            );
+        } else if (hasInitGoal) {
+            datasets.push(
+                generateLine(
+                    [points.kickStarterStart, points.kickstarterEnd],
+                    chartBlue,
+                    'Kickstarter Funded'
+                )
+            );
+        }
+
+        if (hasExceededInitGoal) {
+            datasets.push(
+                generateLine(
+                    [points.curveStart, points.currentSupply],
+                    chartBlue,
+                    'Curve chart funded'
+                )
+            );
+
+            datasets.push(
+                generateLine(
+                    [points.kickstarterEnd, points.curveStart],
+                    chartBlue,
+                    'Curve chart point of change'
+                )
+            );
+
+            datasets.push(
+                generateLine(
+                    [points.currentSupply, points.maxSupplyToShow],
+                    chartGray,
+                    'Curve chart unfunded'
+                )
+            );
+        } else {
+            datasets.push(
+                generateLine(
+                    [points.kickstarterEnd, points.curveStart],
+                    chartGray,
+                    'Curve chart point of change'
+                )
+            );
+
+            datasets.push(
+                generateLine(
+                    [points.curveStart, points.maxSupplyToShow],
+                    chartGray,
+                    'Curve chart funded'
+                )
+            );
+        }
+
+        datasets.push(generateSupplyMarker(points.currentSupply, 'Current Supply'));
+
+        // TODO: Determine how to display predicted future supply / price after current buy
```

Comment:
We probably don't need this as a TODO.  Can add it as a nice to have enhancement under the github issues.

Reply:
I'm going to just include the feature for now - It'll help us think about the best way to display it. Created an issued as well.